src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
```

Comment:
I think it's really good that you broke the different commands into their own classes!